src/Entities/Entity.h

```diff
@@ -504,6 +508,10 @@ class cEntity
 	For realistic effects, this should be negative. For spaaaaaaace, this can be zero or even positive */
 	float m_Gravity;
 	
+	/** Stores the air drag that is applied to the entity every tick, measured in speed ratio per second
+	Acts as air friction and slows down flight */
```

Comment:
`per second` will be quite interesting to implement in the actual gravity code. `per tick` might be a bit easier, although all the values would need recalculating.